src/config/storage/json.rs

```diff
@@ -53,38 +51,45 @@ impl TryFrom<&String> for JsonStorageAdapter {
     }
 }
 
-impl Store for JsonStorageAdapter {
-    fn get_setting(&self, key: &str) -> Option<Setting> {
-        self.elements.get(key).cloned()
+impl StorageAdapter for JsonStorageAdapter {
+    fn get(&self, key: &str) -> Option<Setting> {
+        let lock = self.elements.lock().unwrap();
```

Comment:
We read during a lock. This is fast enough. I don't even know if we really need it here.